_includes/1.0/overview/filter.html

```diff
@@ -0,0 +1,29 @@
+<div id="filter" class="group">
+    <h2>Filter</h2>
+    <p>
+        API users are able to apply conditions on requests that support the filter feature.
+        Pass allowed parameters mentioned in each section named "Available Filter Options" of endpoints that handle this feature. 
+    </p>
+    <p>
+        Thanks to comparison and conjunction operators filter your allow to retrieve and shape data the way that you need it
```

Comment:
Thanks to comparison and conjunction operators filter, you're allowed to retrieve and shape data the way you need.
For instance, to retrieve realized accounting of a certain journal within 2020 first quarter, you can pass the following parameters to the "transations" end point :
?date=gte:2020-01-01%20AND%20lte:2020-03-31&journal_id=4&status=closed